### ISSUE TITLE
Missing unit symbols

### ISSUE THREAD

**Issue by pedroteixeira** [josdejong/mathjs#1082]
Hello,
Just noticed that in v4.1.1:
```
> mathjs.eval('100 years to decades')
Error: Undefined symbol decades
```

But in the docs http://mathjs.org/docs/datatypes/units.html, it does mention the alias:
` decade (decades), century (centuries), millennium (millennia)`

thanks for the great library!

**Comment by josdejong** [comment]
Thanks for reporting Pedro, these three plurals where indeed missing. I've added them, will be available in the first next release.

**Comment by josdejong** [comment]
Fixed in v4.1.2
